### PR TITLE
WIP: script to optimize pngs with imagemin

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

**Comment by XhmikosR** [comment]
You should also specify zopflipng's `transparent` option.

The rest look good. Or sensible enough :P I used many more iterations but the gain is too small for the time spent.

**Comment by znarf** [comment]
@XhmikosR Do you see a need to integrate `pngout`?

**Comment by XhmikosR** [comment]
@znarf: I'd say no for starters, although pngout isn't so slow like zopflipng.

File: scripts/optimize_png.js [hunk 1]
```diff
@@ -12,6 +13,8 @@ const paths = [
   'src/static/images/buttons/',
   'src/static/images/email/',
   'src/static/images/icons/',
+  // 'src/public/images/mime-pdf' // crash with transparent
```
Comment (XhmikosR) [inline scripts/optimize_png.js]: Weird, it works fine for me with my own zopflipng build https://github.com/XhmikosR/zopfli/releases, which I have submitted to imagemin/zopflipng-bin here https://github.com/imagemin/zopflipng-bin/pull/15

**Comment by XhmikosR** [comment]
I don't see why you need to optimize the PNGs yourself.... We have my PR open and you will not be able to achieve the same results like I told you already.

**Pull request closed without merging**